components/camel-google/camel-google-pubsub/src/main/java/org/apache/camel/component/google/pubsub/GooglePubsubProducer.java

```diff
@@ -59,6 +59,9 @@ public GooglePubsubProducer(GooglePubsubEndpoint endpoint) {
     public void process(Exchange exchange) throws Exception {
 
         if (logger.isDebugEnabled()) {
+            // TODO update when basline is java 21
```

Comment:
typo basline --> baseline , I guess it wil lbe useful to have the correct workding to easily search for similar todos